CMakeLists.txt

```diff
@@ -19,7 +19,13 @@ include(GNUInstallDirs)
 
 set(QPACKAGEKIT_API_LEVEL "1")
 
-find_package(QT NAMES Qt6 Qt5 NO_MODULE REQUIRED COMPONENTS Core DBus)
```

Comment:
Where's the find_package() call now?

Reply:
In https://github.com/PackageKit/PackageKit-Qt/blob/main/src/CMakeLists.txt#L3

Reply:
Odd, let's keep the one here and drop the one in src/.

Reply:
the one here is not an ordinary one, note the "NAMES" part.

That said we can move the proper call to the front